docs/UserGuide.md

```diff
@@ -158,23 +158,87 @@ In this guide, the syntax / format of a command is shown like this:
 
 ***
 
-# 4. Features
+# 4. Beginner's tutorial
 
-This section lists the types of commands that TutorAid can execute. For more information about how to interpret these commands, check out the [Command Format](#23-command-format) and [Command Syntax](#231-command-syntax-in-this-guide) sections.
+Welcome to TutorAid's beginner's tutorial! Here, you will learn how you can use some of our commands to manage the details of your students and lessons on a day-to-day basis.
 
-## 4.1 Student Commands
-### Adding a student: `add`
+If this is your first time launching TutorAid, you will be able to see that some sample data has been provided to you. For this tutorial, let's assume that you tutor the students
+shown in the student panel and that you offer the lesson shown in the lesson panel.
```

Comment:
Wonder if you could capitalise Student Panel and Lesson Panel for consistency (I capitalised all the UI elements when I made the About section)